model/trace/http.yaml

```diff
@@ -19,6 +19,10 @@ groups:
       - ref: http.request.method
         sampling_relevant: true
         requirement_level: required
+      - ref: network.peer.address
+      - ref: network.peer.port
+        requirement_level:
+          recommended: If `network.peer.address` is set.
```

Comment:
wonder if we can move this to the `network.peer.port` definition and removing from everywhere else

Reply:
it looks like it works (tooling-wise). is it ok to apply `requirement_level` in the registry tho?

Reply:
Let's keep the registry free from `requirement_level` definitions, please.

Requirement level is always context specific, while the intention of the registry is to keep attribute definitions free from context-specific details.

So, I personally like it that we explicitly define the requirement level per specific context / usage of an attribute and would keep it as is.